app/views/second_level_browse_page/_links.html.erb

```diff
@@ -1,6 +1,14 @@
 <div class="browse__inner <%= page.lists.curated? ? 'browse__inner--curated' : 'browse__inner--alphabetical' %>">
   <h2 tabindex="-1" class="browse__heading js-heading"><%= t("shared.browse.prefix") %> <%= page.title %></h2>
 
+  <%= render "govuk_publishing_components/components/intervention", {
+    suggestion_text: "Help improve GOV.UK",
+    suggestion_link_text: "Take part in user research",
```

Comment:
I'm guessing there aren't translations yet for the `suggestion_text` or `suggestion_link_text` but could this text come from a translations file please?

Reply:
I'd normally agree to do this, but we then have to add keys with missing translations to _all_ of the non-en files as [this test expects](https://github.com/alphagov/collections/blob/main/spec/lib/locales_validation_spec.rb). As the banner is temporary, can we get away with not?

Reply:
Ah, yeah that's a bit of a pain, I hadn't realised that. In that case yes, that's fine.

Reply:
Thank you!